sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
I'm not sure. e.g., after joins of many tables, if `sizeInBytes` is computed by the simple way (non-cbo way), we just multiply all the sizes of these tables, then `sizeInBytes` becomes a ridiculously large value. I think this will harm user experience.
I agree removing the flag can simplify code a lot, but I'm hesitated to expose such information to all users.